pkg/oc/admin/project/new_project.go

```diff
@@ -61,6 +63,9 @@ func NewCmdNewProject(name, fullName string, f *clientcmd.Factory, out io.Writer
 			if options.Client, _, err = f.Clients(); err != nil {
 				kcmdutil.CheckErr(err)
 			}
+			authorizationClient, err := f.OpenshiftInternalAuthorizationClient()
```

Comment:
Seems like most of the logic is this area should be moved into the `complete` func.